test/libsolidity/SolidityExpressionCompiler.cpp

```diff
@@ -221,6 +221,21 @@ BOOST_AUTO_TEST_CASE(int_with_wei_ether_subdenomination)
 	BOOST_CHECK_EQUAL_COLLECTIONS(code.begin(), code.end(), expectation.begin(), expectation.end());
 }
 
+BOOST_AUTO_TEST_CASE(int_with_gwei_ether_subdenomination)
```

Comment:
@chriseth why do we have these test cases still and not just have them replaced with `semanticTests`?

Reply:
These are unit tests that check that the correct code is generated. The semantic tests only check that the generated code has a certain semantic :)

Reply:
I know, but see little point checking that a specific PUSH was generated for these. The semantic tests would check the value too.